App.tsx

```diff
@@ -74,8 +74,10 @@ import {filterLoggedData} from 'logging/filterLoggedData';
 import mixpanel from 'mixpanel';
 import PostHog, {PostHogProvider} from 'posthog-react-native';
 import {startupUpdateCheck, UpdateStatus} from 'Updates';
+import * as Linking from 'expo-linking';
 
 logger.info('App starting.');
+const prefix = Linking.createURL('/');
```

Comment:
note to self: if this is just for supporting Expo Go, maybe they recommend some other approach? code that sits around only for debugging always feels a bit weird

Reply:
I mean we can really just take it out, no need to test this in expo once we know it works via universal links since it doesnt actually help us that much anyways